src/ch02-00-guessing-game-tutorial.md

```diff
@@ -0,0 +1,2062 @@
+# Guessing Game
+# Jogo de Adivinhação
+
+Let’s jump into Rust by working through a hands-on project together! This
+chapter introduces you to a few common Rust concepts by showing you how to use
+them in a real program. You’ll learn about `let`, `match`, methods, associated
+functions, using external crates, and more! The following chapters will explore
+these ideas in more detail. In this chapter, you’ll practice the fundamentals.
+
+Vamos entrar de cabeça no Rust e colocar a mão na massa! Este capítulo vai lhe
+apresentar alguns conceitos bem comuns no Rust, mostrando como usá-los em um
+programa de verdade. Você vai aprender sobre `let`, `match`, métodos, funções
+associadas, crates externos, e mais! Os capítulos seguintes vão explorar essas
```

Comment:
Termos que são jargão Rust (_crate_ é o caso clássico) estrangeiro poderiam ser escritos com _itálico_, ao menos na primeira aparição.

Não sei se isto é uma "norma escrita" do projeto, porém.

Reply:
Então, o próprio texto original em inglês já enfatiza esses termos do jargão Rust usando itálico. Acho que se colocarmos outras palavras em itálico, vamos acabar ofuscando as que realmente deveriam ter mais destaque. Coloquei isso em discussão na issue #42, dá uma olhada e opina lá, também. :)